app/middleware/getServices.js

```diff
@@ -0,0 +1,70 @@
+const VError = require('verror').VError;
+const log = require('../lib/logger');
```

Comment:
Please alphabetise the requires, whilst keeping the external modules listed first.